app/src/main/resources/queries/nearby_query.rq

```diff
@@ -39,6 +39,12 @@ SELECT
        # Get emoji
        OPTIONAL { ?classId wdt:P487 ?emoji0. }
        OPTIONAL { ?classId wdt:P279*/wdt:P487 ?emoji1. }
+
+       OPTIONAL {
+           ?wikipediaArticle   schema:about ?item ;
+                               schema:isPartOf <https://${LANG}.wikipedia.org/> .
+           SERVICE wikibase:label { bd:serviceParam wikibase:language "${LANG}" }
+         }
        OPTIONAL {
            ?wikipediaArticle   schema:about ?item ;
```

Comment:
I guess you can remove this block? Most Wikidata items do not have an English Wikipedia article so it is not really a fallback option either.

Reply:
Now I see that you tested with English as fallback. In a next step later, we might want to fallback to _any_ language if both the user's language and English are not available, but this will do for now :-)